src/utils/misc/readableTime.js

```diff
@@ -1,12 +1,53 @@
+function secondsToMinutes(timeInSeconds) {
+  return Math.floor(timeInSeconds / 60);
+}
+function secondsToHours(timeInSeconds) {
+  return Math.floor(secondsToMinutes(timeInSeconds) / 60);
+}
+function secondsToDays(timeInSeconds) {
+  return Math.floor(secondsToHours(timeInSeconds) / 24);
+}
+
+function getHumanRoundedTime(timeInSeconds, precision = "seconds") {
```

Comment:
i will merge it for now @tfnribeiro , but i guess we can remove this because it's not used anymore.